journal/202210.md

```diff
@@ -0,0 +1,472 @@
+# {DRAFT} Decred Journal – October 2022
+
+{ _Only remove DRAFT from the title when all other todos are solved, at the very end. Search for `{` to see what's left._ }
+
+{ _Hint: Hints just add context. If it doesn't start with "Hint: ", it is a task. This was a hint._ }
+
+{ _Hint: Read the Guidelines on how to make DJ great: https://github.com/xaur/decred-news/blob/docs/guidelines.md ._}
+
+![{alt text}](../img/{file.ext} "{hover tooltip text}")
+
+_Image: { "Title by @author. Description..." OR just "Description..." }_
+
+{ _Month's highlights, no links here, 5-7 bullets max._
+
+- 
+
+- 
+
+- 
+
+- 
+
+- 
+
+}
+
+Contents: { _Sync ToC at the very end, when all text is finished_ }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Ecosystem](#ecosystem)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Discussions](#discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+{ _Add individual sections for high-impact announcements before Development, such as consensus votes, software releases, milestones, major bugs, etc._ }
+
+
+## Core Software v1.7.5 Released
+
+{ highlights, links, signature check reminder, optional image }
+
+https://github.com/decred/decred-binaries/releases/tag/v1.7.5
+
+
+## DCRDEX v0.5.4 Released
+
+{ what was fixed, maybe drop this section and resort to Dev/DCRDEX if too small }
+
+https://github.com/decred/dcrdex/releases/tag/v0.5.4
+
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+
+### dcrd
+
+_[dcrd](https://github.com/decred/dcrd) is a full node implementation that powers Decred's peer-to-peer network around the world._
+
+- Utilize [server context](https://github.com/decred/dcrd/pull/3011) when connecting to peers to make shutdown more responsive. { _Correct?_ }
+- Use [context in RPC tests](https://github.com/decred/dcrd/pull/3012). { _Is it to test that cancellation/shutdown works, or to be able to cancel/abort the test itself?_ }
+- Make sure peer address is [added to address manager](https://github.com/decred/dcrd/pull/3013). { _Is it a bugfix? If so, what was the bug?_ }
```

Comment:
It helps speed up discovery of new nodes on the network by ensuring they are added to the list of addresses that need to be tried in order to build reputation when they haven't otherwise been seen before.

Reply:
Perfect, added as is.